src/generic/stepper/Header.jsx

```diff
@@ -29,16 +28,23 @@ export default function Header({ steps, className }) {
     >
       {steps.map(({ iconLabel, label, incomplete }, index) => {
         const isNotLastStep = index < steps.length - 1;
+        const isActiveStep = (isFirstStep && index === 0) || (!isFirstStep && index === 1);
```

Comment:
if you unpack the step as well, then the following could become
```
{steps.map(({ iconLabel, label, incomplete, step }, index) => { ... 
         isFirstStep && step === index +1 || (!isFirstStep && step === index +1);
```

Reply:
actually, I ideally want to remove this check altogether. and want to have an active step number and do it like this

activeStepNo === index+1; removing dependency of isFirstStep. I wil try implementing it through redux.

Reply:
sounds good.